CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased]
 ### Added
 - CHANGELOG.md
+- PHP 7 type declarations for method parameters and return values
+### Removed
+- Drop support for PHP 5.x and PHP 7.1 as they are no longer
```

Comment:
This is dropping support for PHP 7.0, no?

Reply:
👍 Fixed